v4/export/dump.go

```diff
@@ -120,3 +134,55 @@ func dumpTable(ctx context.Context, conf *Config, db *sql.DB, dbName string, tab
 	}
 	return nil
 }
+
+func concurrentDumpTable(ctx context.Context, conf *Config, db *sql.DB, dbName string, tableName string) (bool, error) {
+	// try dump table concurrently by split table to chunks
+	chunksCh := make(chan *tableDataChunks, defaultDumpThreads)
+	errCh := make(chan error, defaultDumpThreads)
+	skipCh := make(chan struct{})
+
+	ctx1, cancel1 := context.WithCancel(ctx)
+	defer cancel1()
+	var g errgroup.Group
+	g.Go(func() error {
+		splitTableDataIntoChunks(ctx1, chunksCh, errCh, skipCh, dbName, tableName, db, conf)
+		return nil
+	})
+
+Loop:
+	for {
+		select {
+		case <-ctx.Done():
+			return false, nil
+		case chunk, ok := <-chunksCh:
+			if ok {
+				g.Go(func() error {
+					fileName := fmt.Sprintf("%s.%s.%d.sql", dbName, tableName, chunk.offset)
```

Comment:
why not use `writer.WriteInsert`?

Reply:
I suggest you modify the implementation of `Write Table Data(ctx context.Context, ir TableDataIR) error`, and use it uniformly to write data